packages/cid-verifier/scripts/cli.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+import sade from 'sade'
+
+import { buildCmd } from './build.js'
+import { heartbeatCmd } from './heartbeat.js'
```

Comment:
```suggestion
```

we don't need hearbeat. It is used for tracking if cron jobs are running and creating alerts to opsgeanie